slashCommands/vote.ts

```diff
@@ -145,14 +167,20 @@ async function printVotes(pro: Set<string>, con: Set<string>, reply: Message, ti
         msgEmbed.setDescription(title);
     }
     //show embed
-    await reply.edit({embeds:[msgEmbed]})
+    await reply.edit({embeds:[msgEmbed], components:[]})
+}
+
+function membercanstartvote (member: GuildMember, selectedRole: Role): boolean {
```

Comment:
The name of this function is a bit misleading at this point, because it checks if a member can't start a vote, while it would indicate the negation of this by its name